pkg/coinbase/staking_context_balances.go

```diff
@@ -0,0 +1,106 @@
+package coinbase
+
+import (
+	"context"
+
+	"github.com/coinbase/coinbase-sdk-go/gen/client"
+)
+
+// stakingBalances represents the active stakeable balances for a given address and asset.
+type stakingBalances struct {
```

Comment:
Did we wanna rename the type as well to something like `StakingContextBalances` ? currently very similar to `StakingBalance`

Reply:
Yeah thats a good idea!